src/Paginator.php

```diff
@@ -94,12 +94,10 @@ public function setTotal($total)
     /**
      * Determine and return the current page. You can extend this method for
      * the advanced logic.
-     *
-     * @return int
      */
-    public function getCurrentPage()
+    public function getCurrentPage(): int
     {
-        return isset($_GET[$this->urlTrigger]) ? (int) $_GET[$this->urlTrigger] : 1;
+        return $_GET[$this->urlTrigger] ?? 1;
```

Comment:
This is not the same, no (int) casting !!!

Reply:
I done it using the return type ;-)

Reply:
yes please change to `return (int)($_GET.... ?? 1)`

Reply:
oh awesome then!]